src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int NUMBER_BOUND = 101;
+    private final ColorSupplier supplier = new ColorSupplier();
+
+    public Ball getRandomBall() {
+        Random random = new Random();
```

Comment:
Still the issue with random. On each call of getRandomBall() you create a new instance of random, which is bad, lets put this instance on the class level, so we can reuse.